modules/core/src/v2/coins/erc20Token.ts

```diff
@@ -244,16 +244,18 @@ export class Erc20Token extends Eth {
       const encodedArgs = optionalDeps.ethAbi.rawEncode(_.map(sendMethodArgs, 'type'), _.map(sendMethodArgs, 'value'));
       const sendData = Buffer.concat([methodSignature, encodedArgs]);
 
-      // Build contract call and sign it
-      const tx = new optionalDeps.EthTx({
+      const txParams = {
```

Comment:
needs to be extracted because spendAmount isnt an actual parameter for `fromTxData`. Not sure if this parameter is actually needed...

Reply:
Looks like it's not used anywhere else here either, why not just remove it if it's not being used?

Reply:
https://github.com/BitGo/BitGoJS/pull/1408